### PR TITLE
ScrollView displays a horizontal scrollbar incorrectly on window resize when RTL is enabled (T934842)

### PR DESCRIPTION
**Pull request by EugeniyKiyashko**

### CODE REVIEW

**Review by AryamnovEugeniy — DISMISSED** [review]

File: js/ui/scroll_view/ui.scrollable.js [hunk 1]
```diff
@@ -156,17 +150,37 @@ const Scrollable = DOMComponent.inherit({
         this.update();
 
         this.callBase();
+
+        this._rtlConfig = {
+            scrollRight: 0,
+            clientWidth: this._container().get(0).clientWidth,
```
Comment (IgnatovDan) [inline js/ui/scroll_view/ui.scrollable.js:156]: Saving clientWidth value in the '_initMarkup' method looks strange. Usually DOM tree is not ready at this moment.

File: js/ui/scroll_view/ui.scrollable.js [hunk 2]
```diff
@@ -156,17 +150,37 @@ const Scrollable = DOMComponent.inherit({
...
+            windowPixelRatio: this._getWindowDevicePixelRatio()
+        };
         this._updateRtlPosition();
     },
 
+    _isHorizontalRtl: function() {
```
Comment (IgnatovDan) [inline js/ui/scroll_view/ui.scrollable.js]: imho _isHorizontalAndRtlEnabled is better

File: js/ui/scroll_view/ui.scrollable.js [hunk 3]
```diff
@@ -198,6 +212,19 @@ const Scrollable = DOMComponent.inherit({
         eventsEngine.on(this._$container, addNamespace('scroll', SCROLLABLE), strategy.handleScroll.bind(strategy));
     },
 
+    _updateRtlConfig: function() {
+        const config = this._rtlConfig;
```
Comment (IgnatovDan) [inline js/ui/scroll_view/ui.scrollable.js]: imho it's better mention 'this.' at each access instead of a local variable usage

**Review by IgnatovDan — CHANGES_REQUESTED** [review]

File: js/ui/scroll_view/ui.scrollable.native.js [hunk 4]
```diff
@@ -145,6 +145,7 @@ const NativeStrategy = Class.inherit({
 
     handleScroll: function(e) {
         // NOTE: ignoring scroll events when scroll location was not changed (for Android browser - B250122)
+        this._component._updateRtlConfig();
```
Comment (IgnatovDan) [inline js/ui/scroll_view/ui.scrollable.native.js:147]: imho "// NOTE: ignoring scroll events" should be inside the 'if(!this._isScrollLocationChanged()) {'

File: js/ui/scroll_view/ui.scrollable.js (hunk 1 above)
Comment (novsstation) [reply js/ui/scroll_view/ui.scrollable.js:156]: Nope, I create the rtlCnfig in the end of the render function. Github just shows diff in this manner.

File: js/ui/scroll_view/ui.scrollable.js (hunk 2 above)
Comment (novsstation) [reply js/ui/scroll_view/ui.scrollable.js]: Yep, fixed - https://github.com/DevExpress/DevExtreme/pull/14919/commits/9e7177e93126466fe94baa879b005386cbb60bbc

File: js/ui/scroll_view/ui.scrollable.js (hunk 3 above)
Comment (novsstation) [reply js/ui/scroll_view/ui.scrollable.js]: Yes, exactly.
Fixed https://github.com/DevExpress/DevExtreme/pull/14919/commits/9e7177e93126466fe94baa879b005386cbb60bbc

File: js/ui/scroll_view/ui.scrollable.native.js (hunk 4 above)
Comment (novsstation) [reply js/ui/scroll_view/ui.scrollable.native.js:147]: Thank you for your attention.
Fixed https://github.com/DevExpress/DevExtreme/pull/14919/commits/9e7177e93126466fe94baa879b005386cbb60bbc

**Review by IgnatovDan — APPROVED** [review]

**Review by novsstation — APPROVED** [review]

**Pull request merged**